packages/runtime/client/rest/src/withdrawal/endpoints/collection.ts

```diff
@@ -64,27 +54,27 @@ export const getHeadersByRangeViaAxios: (
             )
           : ""
       }`,
-      request && request.range
-        ? { headers: { Range: unWithdrawalsRange(request.range) } }
-        : {}
+      request && request.range ? { headers: { Range: request.range } } : {}
     ),
     TE.map(([headers, data]) => ({
       data: data,
-      previousRange: headers["prev-range"],
-      nextRange: headers["next-range"],
+      page: {
+        current: headers["content-range"],
+        next: headers["next-range"],
+        total: Number(headers["total-count"]).valueOf(),
+      },
```

Comment:
Pagination should be marked in the changelog as breaking change, indicating the removal of previouseRange that was never possible to get.